frontend/src/store/modules/common/actions.ts

```diff
@@ -168,31 +168,31 @@ export default {
     async addRepo(context: any, payload: any) {
         return new Promise((resolve, reject) => {
             setTimeout(() => {
-                let rg_name = payload.rg_name || undefined
-                let repo_name = payload.repo_name || undefined
-                let repo_id = payload.repo_id || undefined
-                let repo_group_id = payload.repo_group_id || undefined
-                let gitURL = payload.url || undefined
+                const rgName = payload.rg_name || undefined
+                const repoName = payload.repo_name || undefined
+                const repoId = payload.repo_id || undefined
+                const repoGroupId = payload.repo_group_id || undefined
+                const gitURL = payload.url || undefined
 
-                let repo: Repo = context.state.AugurAPI.Repo({
-                    gitURL: gitURL,
-                    repo_id: repo_id,
-                    repo_group_id: repo_group_id,
-                    rg_name: rg_name,
-                    repo_name: repo_name
+                const repo: Repo = context.state.AugurAPI.Repo({
+                    '{gitURL}': gitURL,
```

Comment:
@Nebrethar: Do you use some tool to fix lint error automatically? There are some problem about variable name. We don'e need to convert Object key to string and `gitURL` name changed to `{gitURL}`.

Reply:
I did this all by hand. The program requested a string literal, '{gitURL}', instead of a floating name for certain types.